catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -90,6 +94,9 @@ public UpdateRequest process(UpdateRequest input)
                 counter.getOrDefault(Validation.VALIDATION_WARNINGS, 0),
                 counter.getOrDefault(Validation.VALIDATION_ERRORS, 0));
 
+        INGEST_LOGGER.info("{} metacards were removed for failing an enforced validator.",
```

Comment:
I'm going to have to disagree with your disagreement. Having that many log messages in an enterprise environment is going to flood the logs and cause performance degradation.